tests/helpers/test_cherrypicker_data_helper.py

```diff
@@ -0,0 +1,320 @@
+import os
+import shutil
+from collections import namedtuple
+from datetime import datetime
+from unittest.mock import patch
+
+import pytest
+
+from crawler.helpers.cherrypicker_test_data import (
+    create_barcode_meta,
+    create_barcodes,
+    create_csv_rows,
+    create_plate_rows,
+    create_rna_id,
+    create_rna_pcr_id,
+    create_root_sample_id,
+    create_row,
+    create_test_timestamp,
+    create_viral_prep_id,
+    flat_list_of_positives_per_plate,
+    flatten,
+    generate_baracoda_barcodes,
+    write_plates_file,
+)
+from tests.conftest import is_found_in_list
+
+LoggerMessages = namedtuple("LoggerMessages", ["info", "error"])
+
+
+@pytest.fixture
```

Comment:
Shared fixtures should be added to `conftest.py`.

Reply:
It's not actually shared if you look carefully.  The logger is already imported into the `cherrypicker_test_data` by the time we get to patch it, so patching the source at `logging.logger` doesn't work.  The patch has to be done on the specific module that this test file is testing.

Reply:
Gotcha, thanks for the explanation.